python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -677,11 +677,22 @@ def _schedule_first_task(self, schedule, current_time):
 
                 schedule_id = uuid.UUID(row.get('id'))
 
+                #
+                # row.get('schedule_day') returns an int, say 0, from SQLite
+                # and "0", as a string, from Postgres
+                # We handle here this difference
+                #
+
+                if type(row.get('schedule_day')) is str:
+                    s_day = int(row.get('schedule_day')) if row.get('schedule_day').strip() else None
+                else:
+                    s_day = int(row.get('schedule_day'))
+
```

Comment:
Now as we have proper int values https://github.com/foglamp/FogLAMP/pull/729 ; do we still need above workaround? /cc @amarendra-dianomic @pintomax @MarkRiddoch